e2e/tests/omnibus-arbitrum-mainnet.toml/Liquidation_Rewards_Registrations.e2e.js

```diff
@@ -0,0 +1,170 @@
+    log({ supportedCollaterals });
+
+    const liquidatableCollaterals = supportedCollaterals
+      // We do not need rewards distributor for system token
+      .filter((collateral) => collateral.address !== systemToken.address);
+
+    expectedDistributors = [];
+    for (const liquidatableCollateral of liquidatableCollaterals) {
+      const { distributor, poolDelegatedCollateralTypes } = await contractRead({
+        wallet,
+        contract: 'PerpsMarketProxy',
+        func: 'getRegisteredDistributor',
+        args: [liquidatableCollateral.synthMarketId],
+      });
+      for (const collateralType of poolDelegatedCollateralTypes) {
+        expectedDistributors.push({
+          poolId,
+          collateralType,
+          distributor,
+        });
+      }
+    }
+    log({ expectedDistributors });
+  });
+
+  it('should ensure each perps collateral has rewards distributor registered', async () => {
+    for (const expectedDistributor of expectedDistributors) {
+      const registeredDistributor = registeredDistributors.find(
+        ({ poolId, collateralType, distributor }) =>
+          collateralType === expectedDistributor.collateralType &&
+          distributor === expectedDistributor.distributor &&
+          poolId === expectedDistributor.poolId
+      );
+      log({ expectedDistributor, registeredDistributor });
+      assert.ok(
+        registeredDistributor,
+        'Every expected rewards distributor for liquidations is registered'
+      );
+    }
+  });
+
+  it('should create new random wallet', async () => {
+    log({ wallet: wallet.address, pk: wallet.privateKey });
+    assert.ok(wallet.address);
+  });
+
+  it('should set ETH balance to 100', async () => {
+    const address = wallet.address;
+    assert.equal(await getEthBalance({ address }), 0, 'New wallet has 0 ETH balance');
+    await setEthBalance({ address, balance: 100 });
+    assert.equal(await getEthBalance({ address }), 100);
+  });
+
+  it('should create user account', async () => {
+    assert.equal(
+      await getAccountOwner({ accountId }),
+      ethers.constants.AddressZero,
+      'New wallet should not have an account yet'
+    );
+    await createAccount({ wallet, accountId });
+    assert.equal(await getAccountOwner({ accountId }), address);
+  });
+
+  it('should verify that new wallet has no liquidation rewards without "reward is not found" revert', async () => {
+    for (const rewardsDistributor of expectedDistributors) {
+      const collateralType = rewardsDistributor.collateralType;
+      const distributor = rewardsDistributor.distributor;
+      log({ accountId, poolId, collateralType, distributor });
+
+      const availableRewards = await getAvailableRewards({
+        accountId,
+        poolId,
+        collateralType,
+        distributorAddress: distributor,
+      });
+      log({ availableRewards });
+      assert.equal(availableRewards, 0, 'new wallet should have no liquidation rewards');
+    }
+  });
```

Comment:
This test verifies that actual blockchain state has all the expected distributors registered regardless what cannon state tells us.
If for some reason cannon thinks that RD is registered but it was not registered on chain - this test will let as catch it.